projects/canopy/src/lib/button/button.stories.ts

```diff
@@ -30,8 +30,14 @@ const buttonVariants = [
       [size]="size"
       [variant]="variant"
     >
+      <lg-icon *ngIf="icon !== 'None' && doubleIconButton" name="filter" first></lg-icon>
       {{ content }}
-      <lg-icon *ngIf="icon !== 'None'" [name]="icon"></lg-icon>
+      <lg-icon
+        *ngIf="icon !== 'None' && doubleIconButton"
+        [name]="icon"
+        [second]="doubleIconButton"
```

Comment:
Can't `[second]` just be set to true? The `ngIf` means it will always be true, right?

Reply:
yep! thanks

Reply:
Done